modules/migration/migration-resources/5.6.0/dbscripts/step1/identity/db2.sql

```diff
@@ -30,3 +31,5 @@ CREATE TRIGGER SP_CLAIM_DIALECT_TRIG NO CASCADE
 /
 ALTER TABLE SP_CLAIM_DIALECT ADD CONSTRAINT DIALECTID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE
 /
+ALTER TABLE IDN_AUTH_SESSION_STORE ADD EXPIRY_TIME BIGINT AFTER TIME_CREATED;
```

Comment:
As the column is added to the last no need to specify to add after 'TIME_CREATED' column right

Reply:
Fixed